### DISCUSSION TITLE
Looking to add my new Fluid data object in my website

### DISCUSSION THREAD

**Discussion by CoderLama** [microsoft/FluidFramework#6781] [Q&A]
Hi everyone,

I have created a in sync video player using your lovely framework [fluid-coplay](https://github.com/CoderLama/fluid-coplay)

However, I am not sure how do I make it run with my web app. Is there any guideline wrt to web app integration and fluid component publish strategy?

**Comment by tylerbutler** [comment]
@CoderLama Hello! There is now a hosted Fluid service in Azure that you can use to publish Fluid web apps. The documentation here will walk you through deploying a Fluid web app with Azure: <https://docs.microsoft.com/en-us/azure/azure-fluid-relay/how-tos/connect-fluid-azure-service>.

One challenge with deployment is handling authentication. See this section of the docs for details on this: <https://docs.microsoft.com/en-us/azure/azure-fluid-relay/concepts/authentication-authorization>

Please feel free to post here if you have any questions or problems!
